prime-router/metadata/hl7_mapping/ORU_R01/base/message-header.yml

```diff
@@ -19,8 +19,8 @@ elements:
     hl7Spec: [ MSH-4-1 ]
 
   - name: sending-facility-universal-id
-    condition: '%resource.sender.resolve().identifier.value is Oid'
-    value: [ '%resource.sender.resolve().identifier.value.toString().replace(''urn:oid:'','''')' ]
+    condition: '%resource.sender.resolve().identifier[0].value.startsWith(''urn:oid:'')'
```

Comment:
Is it possible to pull this without counting on it being the first element? I will test on my end as well